src/components/Avatar/Avatar.jsx

```diff
@@ -0,0 +1,22 @@
+import './Avatar.scss';
+
+import React, { PureComponent, PropTypes } from 'react';
+
+import cxHelpers from 'lib/decorators/classNameHelpers';
+import Icon from 'components/Icon';
+
+@cxHelpers("Avatar")
+class Avatar extends PureComponent {
+  static propTypes = {
+    uri: PropTypes.string
+  };
+
+  render(){
+    const { uri } = this.props;
+    return uri
```

Comment:
haha I have had this debate back and forth. The Java devs at Gradle brought me around on this; I actually think it reads cleaner now. Happy to change it though, I don't feel strongly.